boring-sys/Cargo.toml

```diff
@@ -24,12 +24,27 @@ include = [
     "/deps/boringssl/LICENSE",
     "/build.rs",
     "/src",
+    "/patches",
+    "/scripts"
 ]
 
-[build-dependencies]
-bindgen = { version = "0.65.1", default-features = false, features = ["runtime"] }
-cmake = "0.1"
+[package.metadata.docs.rs]
+features = ["rpk", "post-quantum"]
```

Comment:
I wouldn't call it `post-quantum`: BoringSSL already supports PQ. More importantly, BoringSSL already support the key agreement that we want everyone to use.

This just adds extra key agreements we don't want others to use. (We need it temporarily for compliance/testing.)

What about `internal-pq` and a note that people shouldn't use it?

Reply:
would like to avoid stuff that is "internal", could it be "experimental-pq" and could you provide a short summary, so we can explain what's "in the package" and why it's experimental?

Reply:
```
Upstream BoringSSL support the post-quantum hybrid key agreement X25519Kyber768Draft00. Most users should stick to that one. Enabling this feature, adds a few other post-quantum key agreements:

- X25519Kyber768Draft00Old is the same as X25519Kyber768Draft00, but under its old codepoint.
- X25519Kyber512Draft00. Similar to X25519Kyber768Draft00, but uses level 1 parameter set for Kyber. Not recommended. It's useful to test whether the shorter ClientHello upsets fewer middle boxes.
- P256Kyber768Draft00. Similar again to X25519Kyber768Draft00, but uses P256 as classical part. It uses a non-standard codepoint. Not recommended.
```